sources/packages/backend/libs/test-utils/src/mocks/ssh-service-mock.ts

```diff
@@ -19,6 +19,12 @@ export const MSFAA_FULL_TIME_RECEIVE_FILE_WITH_REACTIVATION_RECORD =
 export const MSFAA_FULL_TIME_RECEIVE_FILE_WITH_SINGLE_CANCELLATION_RECORD =
   "msfaa-full-time-receive-file-with-single-cancellation-record.dat";
 
+// SFAS received file mocks.
+export const SFAS_LEGACY_RESTRICTION_FILENAME =
```

Comment:
Just a question. The file names actually belong to tests written as it is a test data, and what is the intention of having it delcared in `ssh-service-mock`?

Reply:
I can move it to the test file rather than exporting it from `ssh-service-mock.ts`. I was just following how it was done for the MSFAA test files.

Reply:
Moved the constants to the test file.